ota-plus-web/app/reactapp-new/src/App.tsx

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
 import Routes from './Routes';
 
 import '../style/index.scss';
 
 const App: React.FC<{}> = () => {
-  const [t, i18n] = useTranslation();
+
+  const [data, setData] = useState('')
+
+  const [t] = useTranslation();
+
+  useEffect(() => {
```

Comment:
This is not needed here. Please remove it.

Reply:
@koshelev suggested to leave it for now

Reply:
Please rebase to the newest changes on master. There is redux implemented and possibility of manual sending http request. In this case you can remove your changes in this place.